src/tools/genpolicy/genpolicy-advanced-command-line-parameters.md

```diff
@@ -57,17 +57,29 @@ To enable caching, use the `-u` command line parameter - e.g.,
 $ RUST_LOG=info genpolicy -u -y test.yaml
 ```
 
-# Use containerd to pull and manage images
-You may specify `-d` to use existing `containerd` installation as image manager. This method supports a wider set of images (e.g., older images with `v1` manifest). Needs `sudo` permission to access socket - e.g.,
+# Use containerd to pull and manage images (required for managed identity based authentication)
+
+Prereq: This features needs to run the following script to adapt your docker and containerd config (needs `sudo` access):
```

Comment:
I think these changes are too intrusive to push average users to use this script.

For average users, I would prefer a list of steps in an MD file - similar to the steps from https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/containerd-kata.md. Also, I would like to update such doc Upstream instead of pushing just MSFT users in this direction.

But let's talk with other folks in our team too - maybe they have different opinions.

Reply:
I agree it's too much to require them to use this. This might exist for users that don't feel like doing all these steps, but they are also welcome to do them on their own. I'll add better instructions to the MD, and open a PR upstream with all squashed changes

Reply:
I disagree with: 
1. Calling this script a prerequisite. We should say something like: "if you need help with your containerd settings, see this doc for related tips".
2. Hiding a bunch of details in a script. Users that want to perform these steps should review the doc and perform just those steps that look reasonable to them.